### PR TITLE
[IMP][10.0] medical_patient_dob: Forward port

### PR DESCRIPTION
**Pull request by lasley**
Forward port OCA/vertical-medical#84 & implement OCA/vertical-medical#131 fix

Depends:

- [x] #34

### CODE REVIEW

**Comment by lasley** [comment]
Rebased; ready for review

File: medical_patient_dob/models/medical_patient.py [hunk 1]
```diff
@@ -9,19 +9,23 @@ class MedicalPatient(models.Model):
...
     @api.multi
-    def _format_birthdate_date(self):
+    def _format_dob(self):
         self.ensure_one()
         date = 'No DoB'
-        if self.birthdate_date:
-            date = fields.Datetime.from_string(self.birthdate_date).strftime(
-                '%m/%d/%Y'
+        if self.dob:
+            ResLang = self.env['res.lang']
```
Comment (tedsalmon) [inline medical_patient_dob/models/medical_patient.py:16]: What's with the weird casing here?

**Review by tedsalmon — COMMENTED** [review]
Just have a small question about variable name casing before I approve :)

File: medical_patient_dob/models/medical_patient.py (hunk 1 above)
Comment (lasley) [reply medical_patient_dob/models/medical_patient.py:16]: Indicates Model vs Recordset. Class vs Instance.

**Review by tedsalmon — APPROVED** [review]

**Pull request merged**